### PR TITLE
Add lightning-flash

### PR DESCRIPTION
**Pull request by weiji14**
Crossref https://github.com/PyTorchLightning/lightning-flash/issues/659.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/lightning-flash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+about:
+  home: https://github.com/PyTorchLightning/lightning-flash
+  summary: Flash is a framework for fast prototyping, finetuning, and solving most standard deep learning challenges
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - AddYourGitHubIdHere
```
Comment (weiji14) [inline recipes/lightning-flash/meta.yaml:51]: @Borda and @ethanwharris, are you both ok to be listed as maintainers for the lightning-flash conda recipe? I can also add others in.

```suggestion
    - Borda
    - ethanwharris
    - weiji14
```

**Comment by weiji14** [comment]
@conda-forge/help-python, the Windows build is failing, because `pytorch` does not have Windows packages on the `conda-forge` channel I think (but there are win-64 packages on the `pytorch` channel). Is this ok though, since the package is for a no-arch build?

```
Solving environment: ...working... failed

Leaving build/test directories:
  Work:
 C:\Miniconda\conda-bld\work 
  Test:
 C:\Miniconda\conda-bld\test_tmp 
Leaving build/test environments:
  Test:
source activate  C:\Miniconda\conda-bld\_test_env 
  Build:
source activate  C:\Miniconda\conda-bld\_build_env 


Traceback (most recent call last):
  File "C:\Miniconda\lib\site-packages\conda_build\environ.py", line 804, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {"pytorch[version='>=1.7']"}
##[error]Cmd.exe exited with code '1'.
```

**Comment by Borda** [comment]
@weiji14 maybe you can add `pytorch` channel :rabbit: 
cc: @ethanwharris

**Comment by weiji14** [comment]
> @weiji14 maybe you can add `pytorch` channel rabbit

I don't actually know how to add the channel tbh :upside_down_face: The Windows failures might be acceptable though, since this is a cross-platform noarch build (see https://github.com/conda-forge/staged-recipes/pull/13866#discussion_r593333546).

**Review by dopplershift — APPROVED** [review]
We can ignore the failing Windows build (due to missing dependency) since this is noarch.

**Pull request merged**